conftest.py

```diff
@@ -31,6 +33,10 @@ def pytest_configure(config):
         # DEV: `ext` will contain the `.`, e.g. `.xml`
         config.option.xmlpath = "{0}.{1}{2}".format(fname, os.getpid(), ext)
 
+    # Save per-interpreter benchmark results.
+    if config.pluginmanager.hasplugin("benchmark"):
```

Comment:
we want this only for `CI=true`?

Reply:
I'd thought so. Locally one might just look at the more readable pytest output.

Reply:
👍🏻 I sort of figured the same for `junit.xml`